testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/deployment/OidcWithDeploymentConfigTest.java

```diff
@@ -181,6 +190,44 @@ public static WebArchive createCorsProviderUrlDeployment() {
                 .addAsWebInfResource(OidcWithDeploymentConfigTest.class.getPackage(), CORS_WITH_PROVIDER_URL_FILE, "oidc.json");
     }
 
+    @Deployment(name = SINGLE_SCOPE_APP, managed = false, testable = false)
+    public static WebArchive createSingleScopeDeployment() {
+        return ShrinkWrap.create(WebArchive.class, SINGLE_SCOPE_APP + ".war")
+                .addClasses(SimpleServlet.class)
+                .addClasses(SimpleSecuredServlet.class)
```

Comment:
I think we can remove `SimpleSecuredServlet` here and below since the scope related applications use the new servlet instead.

Reply:
I had to keep it because otherwise, I was getting a 404 error with just the other servlet.

Reply:
I think it should be possible to remove it. You'll likely need to tweak the path that's used to access the servlet and/or the path that's configured for the servlet. 

Take a look at the path that's used for the existing tests when logging into the app and the path that's configured for the existing servlet. I think that should help identify the changes you'll need.

Reply:
I will look into this and update the code. 
All other comments are implemented.